model/CHANGELOG.md

```diff
@@ -2,6 +2,92 @@
 
 Changelog for `twilight-model`.
 
+## [0.5.0] - 2021-06-13
+
+### Upgrade Path
+
+Remove references to `Guild::nsfw`, `Guild::region`, `PartialGuild::nsfw`,
+`PartialGuild::region`, and `TemplateGuild::region`.
+
+Replace the following usages:
+```diff
+-twilight_model::channel::invite::TargetUserType
++twilight_model::channel::invite::TargetType
+
+-twilight_model::gateway::payload::update_status::UpdateStatus
++twilight_model::gateway::payload::update_presence::UpdatePresence
+
+-twilight_model::gateway::payload::update_status::UpdateStatusInfo
++twilight_model::gateway::payload::update_presence::UpdatePresencePayload
+```
+
+### Additions
+
+Support for Slash Commands has been added.
+
+The following models have been added:
+
+- `enum CommandDataOption`
+- `enum CommandOption`
+- `enum CommandOptionChoice`
+- `enum CommandOptionType`
+- `enum CommandPermissionsType`
+- `enum Interaction`
+- `enum InteractionResponse`
+- `enum InteractionType`
+- `enum ReponseType`
+- `struct ApplicationCommand`
+- `struct BaseCommandOptionData`
+- `struct CallbackData`
+- `struct ChoiceCommandOptionData`
+- `struct Command`
+- `struct CommandData`
+- `struct CommandId`
+- `struct CommandInteractionDataResolved`
+- `struct CommandPermissions`
+- `struct GuildCommandPermissions`
+- `struct InteractionChannel`
+- `struct InteractionCreate`
+- `struct InteractionId`
+- `struct InteractionMember`
+- `struct MessageInteraction`
+- `struct OptionsCommandOptionData`
+- `struct PartialApplication`
+- `struct Ping`
+
+### Enhancements
+
+The following models have been updated:
+
+- `bitflags MessageFlgas`: added `EPHEMERAL`
+- `enum Event`: added `InteractionCreate`
+- `enum WebhookType`: added `Application`
+- `struct Message`: added `application_id`, `interaction`
```

Comment:
Same with this, maybe leave out the struct and type the FQN? should be clear what it's referring to